src/config/types.ts

```diff
@@ -219,10 +219,13 @@ type LayerStyleProps = {
 export class BoundaryLayerProps extends CommonLayerProps {
   type: 'boundary';
   path: string; // path to admin_boundries.json file - web or local.
+  adminLevel: number;
   adminCode: string;
   adminLevelNames: string[]; // Ordered (Admin1, Admin2, ...)
   adminLevelLocalNames: string[]; // Same as above, local to country
   styles: LayerStyleProps; // Mapbox line and fill properties.
+  @optional
```

Comment:
do we really need this?

Reply:
Yes. This is to align with `RBD` request to have unique `boundaries` per `admin_level_data` layer. The scenario is that by default if `visibility` is not set we imply it as `true` by default to avoid forcing it on all boundaries in all countries. There are cases where we may have a non-default `boundary` which will be visible for a specific `admin_level_data` layer thus such boundary will have `"visibility": false`

Reply:
e.g.
```json
{

Reply:
why not just assume that you want it visible when you are loading the `x_admin_level_data` that requests it, but just use the default admin_boundary otherwise?